standalone/instructlab-rhoai.md

```diff
@@ -5,330 +5,651 @@ This file documents an experimental install of InstructLab on Red Hat OpenShift
-      JUDGE_CA_CERT:            # Configmap containing CA cert for the judge model (optional - required if using custom CA cert), Example - `kube-root-ca.crt`
-      JUDGE_CA_CERT_CM_KEY:     # Name of key inside configmap (optional - required if using custom CA cert), Example - `ca.crt`
-    ```
-
-* If using a custom CA certificate you must provide the relevant data in a ConfigMap. The config map name and key are then provided as a parameter to the standalone.py script as well as in the `judge-serving-details` secret above.
-
-### Step 3 (Optional) - Setup NFS StorageClass
+If you already have these models deployed you can skip the deployment steps and go straight to the secret set up for
+Judge and Teacher respectively.
+
+[mixtral-8x7b-instruct-v0-1]: https://catalog.redhat.com/software/containers/rhelai1/mixtral-8x7b-instruct-v0-1/6682619da4ea27a10a36e4c6
+[skills-adapter-v3:1.2]: https://catalog.redhat.com/software/containers/rhelai1/skills-adapter-v3/66a89d60ac8385376a3dabed
+[knowledge-adapter-v3:1.2]: https://catalog.redhat.com/software/containers/rhelai1/knowledge-adapter-v3/66a89ce222c498e45a8e0042
+[prometheus-8x7b-v2-0 model]: https://catalog.redhat.com/software/containers/rhelai1/prometheus-8x7b-v2-0/6682611224ba6617d472f451
+
+#### Deploy a judge model server (optional)
+
+Create a service account to be used for token authentication
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <model-server-service-account-name>
+  namespace: <data-science-project-name/namespace>
+```
+
+Upload [prometheus-8x7b-v2-0 model] (Judge-Model) to the same object storage as before.
+
+For example using `ilab` to download and `s3cmd` to sync to object store you can do:
+```bash
+# You can also use Oras or Skopeo cli tools to download the model
+ilab model download --repository docker://registry.redhat.io/rhelai1/prometheus-8x7b-v2-0 --release 1.2
+
+# Default cache location for ilab model download is ~/.cache/instructlab/models
+s3cmd sync path/to/model s3://your-bucket-name/judge-model/
+```
+
+Navigate to the OpenShift AI dashboard
+* Choose Data Science Projects from the left hand menu and choose your data science project/namespace.
+* Choose the data connections tab, and click on the Add data connection button. Enter the details of your S3 bucket (object store) and click Add data connection.
+
+> [!NOTE]
+> Note: Before following the next step - Ensure that the `CapabilityServiceMeshAuthorization` status is `True` in `DSCinitialization` resource.
+
+Create a model server instance
+* Navigate to Data Science Projects and then the Models tab
+* On the right hand side select ‘Deploy model’ under Single-model serving platform
+* Under Serving runtime choose the serving runtime `vLLM Serving Runtime for Kserve`.
+* Check the `Make deployed models available through an external route` box.
+* Under token authentication check the `Require token authentication` box, select the service account that we have created above.
```

Comment:
```suggestion
* Under token authentication check the `Require token authentication` box, write the name of the service account that we have created above.
```

IIRC the there's no selection mechanism here, and Dashboard will create/manage a new SA if not typed to match